src/openforms/utils/tests/test_sanitizer.py

```diff
@@ -0,0 +1,92 @@
+import tempfile
+
+from django.test import TestCase
```

Comment:
consider using `SimpleTestCase` instead, which skips all database initialization and is faster :)